app/profiles/landlord/page.tsx

```diff
@@ -0,0 +1,91 @@
+import { createClient } from "@/utils/supabase/server"
+import { cookies } from "next/headers"
+import { NextPage } from "next"
+
+const landlordProfilePage: NextPage = async ({
+    searchParams
+}: {
+    searchParams?: {
+        landlordId?: string,
+    }
+}) => {
```

Comment:
I think think this is a case where we would be better off using a [dynamic route](https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes) instead of search params. But it doesn't make a huge difference.